distutils/command/install.py

```diff
@@ -190,6 +190,8 @@ class install(Command):
 
     negative_opt = {'no-compile' : 'compile'}
 
+    # Allow Fedora to add components to the prefix
+    _prefix_addition = ""
```

Comment:
I'd also consider something like:

```suggestion
    _prefix_addition = getattr(sysconfig, '_prefix_addition', "")
```

And bypass the need for `_distutils_system_mod`.

Reply:
I wonder whether it would need to be a function, or if a constant works. I guess a constant *should* work, but I would need to double check.

Reply:
I guess it comes down to - is this value something that is static or determinable at startup time, or if it might need to change at run time based on other factors. If you're happy enough with the draft implementation, I can get that into a Setuptools release and you can experiment with a live release (using SETUPTOOLS_USE_DISTUTILS=local).

Reply:
Don't put anything into the release just yet, please. I can test this even without a release, I am just not as fast as you expect me to be (I do other things as well).

Reply:
Happy to oblige. I'm not trying to rush you; only to help. Thanks.

Reply:
Thanks. I expect to do some testing this week.